np.sh

```diff
@@ -1,8 +1,18 @@
 #!/usr/bin/env sh
+
+if test -n "$(git status --porcelain)"; then
```

Comment:
Okay I've pushed up the changes, including this one, though I could see an argument to be made either way.
Either perhaps I forgot to commit something, and it shouldn't publish, or I don't care if it doesn't.
Thinking about it though, does `npm publish` only publish that which is checked into git? I could be wrong, but I thought it would publish anything in the tree, which, if I haven't committed it, should it really be published?

Like I said, though, I could be wrong, so if you want to keep it, I can live with it, but if you want me to revert, I can do that as well.